@here/harp-mapview/lib/copyrights/DefaultCopyrightProvider.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { GeoBox } from "@here/harp-geoutils";
+import { CopyrightInfo } from "./CopyrightInfo";
+import { CopyrightProvider } from "./CopyrightProvider";
+
+/**
+ * Copyright provider which returns static HERE copyright with current year.
+ */
+export class DefaultCopyrightProvider extends CopyrightProvider {
```

Comment:
This should be called HERECopyrightProvider or DefaultHERECopyrightProvider

Reply:
According to Zbigniew's comment, I'll remove `DefaultCopyrightProvider`.